src/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -107,13 +143,56 @@ class AppDelegate: FlutterAppDelegate { // More on the FlutterAppDelegate.
 
 ### Show a FlutterViewController with your FlutterEngine
 
+{% samplecode vc %}
+
+{% sample SwiftUI %}
+The following example shows a generic `ContentView` with a
+`Button` hooked to present a [`FlutterViewController`][].
+The `FlutterViewController` uses the `FlutterEngine` instance
+passed in as an `EnvironmentObject`.
+<!--code-excerpt "ContentView.swift" title-->
+```swift
+import SwiftUI
+import Flutter
+
+struct ContentView: View {
+  // Flutter engine is passed in as an EnvironmentObject.
```

Comment:
similar here, technically it's the flutter dependencies, not flutter engine.

Reply:
Thanks!